### PR TITLE
9 available pools table

### PR DESCRIPTION
**Pull request by pixelPepe**

### CODE REVIEW

**Comment by pixelPepe** [comment]
image

File: .gitmodules
Comment (nadyasav) [inline .gitmodules:1]: Delete [submodule "lib/safe-contracts"] and [submodule "lib/openzeppelin"]

File: lib/openzeppelin
Comment (nadyasav) [inline lib/openzeppelin:1]: Delete

File: lib/safe-contracts
Comment (nadyasav) [inline lib/safe-contracts:1]: Delete

File: lib/openzeppelin
Comment (nadyasav) [reply lib/openzeppelin:1]: Add [forge-std @ f73c73d](https://github.com/foundry-rs/forge-std/tree/f73c73d2018eb6a111f35e4dae7b4f27401e9421)

File: .gitmodules
Comment (pixelPepe) [reply .gitmodules:1]: done

**Review by nadyasav — APPROVED** [review]

**Pull request merged**